app/assets/javascripts/lookup_keys.js

```diff
@@ -187,9 +187,11 @@ function keyTypeChange(item) {
 
 function mergeOverridesChanged(item) {
   var fields = $(item).closest('.fields');
-  var keyType = fields.find("[id$='_key_type']").val();
+  var keyType = fields.find("[id$='_key_type']").find("option").parent().val();
```

Comment:
It's not actually, just found a bug while testing and I fixed it to test my changes. 
I'll remove it from here and open it in a separate PR.
The key_type changed to select2 so now `val()` doesn't actually return the value so the checkboxes would be disabled until you saved the parameter as an array/hash.